app/presenters/contributions/index.rb

```diff
@@ -53,23 +53,23 @@ def stories_table_head_data
       ]
     end
 
-    def stories_table_row_data
-      @stories.map do |story|
+    def contributions_table_row_data
+      @contributions.map do |contribution|
         {
-          id: story.id,
-          url: edit_migration_path(story.id), # TODO: make this campaign-agnostic
-          cells: story_table_row_data_cells(story)
+          id: contribution.id,
+          url: edit_migration_path(contribution.id), # TODO: make this campaign-agnostic
+          cells: contribution_table_row_data_cells(contribution)
         }
       end
     end
 
-    def story_table_row_data_cells(story)
+    def contribution_table_row_data_cells(contribution)
       [
-        story.ore_aggregation.edm_aggregatedCHO&.dc_contributor_agent&.foaf_name,
-        story.ore_aggregation.edm_aggregatedCHO&.dc_identifier,
-        story.created_at,
-        story.aasm_state,
-        story.has_media? ? '✔' : '✘'
+        contribution.ore_aggregation.edm_aggregatedCHO&.dc_contributor_agent&.foaf_name,
+        contribution.ore_aggregation.edm_aggregatedCHO&.dc_identifier,
+        contribution.created_at,
+        t(contribution.aasm_state, scope: 'contribute.contributions.states'),
```

Comment:
this comes out as: `translation missing: en.contribute.pages.contributions.index.published`